lib/util/url.spec.ts

```diff
@@ -81,14 +81,20 @@ describe('util/url', () => {
 
   it('ensures path prefix', () => {
     expect(ensurePathPrefix('https://index.docker.io', '/v2')).toBe(
-      'https://index.docker.io/v2'
+      'https://index.docker.io/v2/'
```

Comment:
Does this change work?

Reply:
It seems to, yes. Though that may be because with the way this function is used, there's always something after the prefix so it's kind of a wash.

I originally had something in this that would check if `path` was `/` and just not include it after the prefix. I wasn't sure which way was preferable, and so I went with this way by default just because it was simpler.